File: C++/3.cpp
/*
Faça um algoritmo que solicita ao usuário um valor inteiro e exibe uma mensagem informando se o número é par ou ímpar.
*/

#include <iostream>

using namespace std;

int main(){
    int numero, resto;

    cout << "Digite um numero inteiro: ";
    cin >> numero;

    resto = numero % 2;

    if(resto == 0){
        cout << "O numero digitado eh par.";
    }else{
        cout << "O numero digitado eh impar.";
        }

    return 0;    
}

File: Python/3.py
"""
Faça um algoritmo que solicita ao usuário um valor inteiro e exibe uma mensagem informando se o número é par ou ímpar.
"""

numeroSolicitado = int(input("Digite um número inteiro: "))

imparPar = numeroSolicitado % 2

if imparPar == 0:
    print("O número informado é par.")
else:
    print("O número informado é ímpar.")
